components/apimgt/org.wso2.carbon.apimgt.rest.api/src/main/java/org/wso2/carbon/apimgt/rest/api/impl/ApisApiServiceImpl.java

```diff
@@ -38,8 +40,6 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import static org.wso2.carbon.apimgt.rest.api.impl.MappingUtil.fromAPItoDTO;
-import static org.wso2.carbon.apimgt.rest.api.impl.MappingUtil.fromDTOtoAPI;
 
 public class ApisApiServiceImpl extends ApisApiService {
```

Comment:
Please note that class comments are available in the up-to-date source in https://github.com/wso2/carbon-apimgt/blob/release-1.10.x/components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/impl/ApisApiServiceImpl.java